src/environment_test.go

```diff
@@ -21,3 +22,30 @@ func TestHostNameWithLan(t *testing.T) {
 	cleanHostName := cleanHostName(hostName)
 	assert.Equal(t, "hello", cleanHostName)
 }
+
+func TestPathWithDriveLetter(t *testing.T) {
```

Comment:
Our main test cases are the other way around. See [this](https://github.com/JanDeDobbeleer/oh-my-posh/blob/d96cb7989a759996ae75e5ee45b95d6b798a7f96/src/segment_crystal_test.go#L10) example.

Reply:
What do you mean "other way around"?

Reply:
I'm guessing I should incorporate the tests inside segment_path_test.go instead of doing this in environment_test.go. Given that there are already some similar tests there.

Reply:
I'm having trouble running the tests. I'm getting the wrong answer for drive letters. It seems as if the drive letters are being left alone in the tests.

This is my test so far (that is ⅔ / 67% failing):

```go
func TestWritePathInfoWindowsOtherDriveLetters(t *testing.T) {
	home := homeBillWindows
	want := "C: > Windows"
	got := testWritePathInfo(home, "C:\\Windows\\", "\\")
	assert.Equal(t, want, got)

	want = "C: > Windows"
	got = testWritePathInfo(home, "c:\\Windows\\", "\\")
	assert.Equal(t, want, got)

	want = "P: > Other"
	got = testWritePathInfo(home, "p:\\Other\\", "\\")
	assert.Equal(t, want, got)
}
```

The 2nd and 3rd test are failing (but they were failing even before my patch)

Reply:
I took your changes and all tests are green on my side.